### DISCUSSION TITLE
記事を「投稿する」ボタンがスクリーンリーダーで押せない

### DISCUSSION THREAD

**Discussion by mo29cg** [increments/qiita-discussions#399] [要望・アイデア・不具合]
投稿する→記事と進んだ後の、
` https://qiita.com/drafts/*/edit# `
上記のURLで「Qiitaに投稿」ボタンを押した後に出る、
「Qiitaに投稿」というモーダルウインドウにスクリーンリーダーでフォーカスを合わせる事ができず、
「投稿する」ボタンも押せません。

試したスクリーンリーダーはNVDA 2022.3.3jpです。

モーダルウインドウのWAI-ARIA周りの実装が何かしら上手くいっていない物と思われます。
視覚障害者が技術記事を書く障害になっているので是非治してほしいです。よろしくお願いします。

**Comment by tanaka-asami415** [comment]
@mo29cg 
ご報告ありがとうございます！
ご不便をおかけして、申し訳ありません🙇こちら調査いたします！

**Reply by mo29cg** [reply]
ありがとうございます！よろしくお願いします！

**Reply by tanaka-asami415** [reply]
@mo29cg
ご報告ができておらず、大変申し訳ございません🙏

上記でご報告いただいた内容ですが、今年の1月に修正が完了いたしました。
現在はすでにスクリーンリーダーでも反応するようになっています！
何かあればまたフィードバックをいただけますと幸いです🙇
